public/css/api.main.css

```diff
@@ -164,3 +164,140 @@ body, html{
     src: url('../fonts/Roboto-Medium.ttf');
     font-size: 18px;
 }
+
+/*
+# myApp Styles
+===============================================
+*/
+
+#main-one {
+    clear: both;
+    min-height: 400px;
+    width: 80%;
+    margin-bottom:20px;
+    background-color: #fff;
+    margin-right: auto;
```

Comment:
**Code Quality - NitPick**
@andela-araimi Make this CSS file more dry, by making use of the margin only declaration.